samples/stock/demo/step-line/demo.js

```diff
@@ -1,28 +1,71 @@
 (async () => {
+    Highcharts.Templating.helpers.lastValue = function () {
+        const points = arguments[0].ctx.points;
+        return points[points.length - 1].y.toFixed(2);
+    };
 
     // Load the dataset
     const data = await fetch(
-        'https://demo-live-data.highcharts.com/aapl-c.json'
+        'https://cdn.jsdelivr.net/gh/highcharts/highcharts@a9dcb12aad/samples/data/investment-simulator.json'
     ).then(response => response.json());
 
     // Create the chart
     Highcharts.stockChart('container', {
-
         rangeSelector: {
-            selected: 1
+            selected: 4
         },
 
         title: {
-            text: 'AAPL Stock Price'
+            text: 'Investment simulator'
         },
 
-        series: [{
-            name: 'AAPL Stock Price',
-            data: data,
-            step: true,
-            tooltip: {
-                valueDecimals: 2
+        yAxis: {
+            labels: {
+                align: 'left',
+                format: '{value:,.2f}'
+            }
+        },
+
+        legend: {
+            enabled: true
+        },
+
+        tooltip: {
+            valueSuffix: ' EUR'
+        },
+
+        plotOptions: {
+            series: {
+                tooltip: {
+                    valueDecimals: 2
+                },
+                pointStart: '2023-01-01',
+                pointInterval: 86400000 // One day
```

Comment:
Using `pointIntervalUnit` is generally safer/better practice, as it will adjust to DST in local time zones.
```suggestion
                pointIntervalUnit: 'day'
```